### PR TITLE
slight rework of text

### PR DESCRIPTION
**Pull request by lazaridiscom**
(1st iteration, few more coming in future, if this one is ok and merged)

removed some duplicated text, slightly simplified text.

- [x] my changes to the code do not affect any exposed aspect of the developer experience

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -93,7 +84,7 @@ Please open a dedicated branch for the change in isolation so we can discuss the
 
 Please broadcast the proposal in chat to maximise visibility and the opportunity for everyone to respond.
 
-It is NOT OK to change the behaviour of tests/CI in otherwise unrelated PRs. SOMETIMES it MAY be OK to change CI in a related PR, e.g. adding a new lib that your code requires. DO expect that a change like this will probably attract additional scrutiny during the PR review process, which is unfortunate but important.
+It is not(!) OK to change the behaviour of tests/CI in otherwise unrelated PRs. Sometimes(!) it MAY be OK to change CI in a related PR, e.g. adding a new lib that your code requires. DO expect that a change like this will probably attract additional scrutiny during the PR review process, which is unfortunate but important.
```
Comment (thedavidmeister) [inline CONTRIBUTING.md]: skim readers often miss the word "not" so i like to capitalise it, the other caps in this paragraph can probably be sentence cased :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -103,76 +94,34 @@ The continuous integration (CI) suite executes the same `. docker/run-test` comm
...
-If your changes do not break the current environment, you can submit a separate Pull Request first, and once it is merged, the CI environment should be up-to-date for your code change Pull Request.
-
-Otherwise, you will need to speak to an admin who can force merge your full changes after testing locally.
-
-The continuous integration (CI) suite executes the same `. docker/run-test` command that developers are encouraged to run.
-
-What happens if I need to change that environment? E.g. what if I need a new system library dependency installed?
-
-- Step 1 - Add the dependency to `docker/Dockerfile.ubuntu`
+1. Add the dependency to `docker/Dockerfile.ubuntu`
```
Comment (thedavidmeister) [inline CONTRIBUTING.md]: hmmmm i think all these docker files in `docker/` are stale :thinking:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -103,76 +94,34 @@ The continuous integration (CI) suite executes the same `. docker/run-test` comm
...
-This section describes our practices and guidelines for using git and making changes to the repo.
+## Repository Related Processes
 
 * We use Github's pull requests as our code review tool
 * We encourage any dev to comment on pull requests and we think of the pull request not as a "please approve my code" but as a space for co-developing, i.e. asynchronous "pair-coding" of a sort.
 * We develop features on separate branches identified by the Github issue number, i.e. `124-my-new-feature`
 * We use merge (not rebase) so that commits related to a ticket can be retroactively explored.
 * In most repos development happens on a `develop` branch which gets merged to master when there's a release.
 
-## License
```
Comment (thedavidmeister) [inline CONTRIBUTING.md]: the licence info is important, especially the reference to that medium article

**Comment by lazaridiscom** [comment]
@thedavidmeister , seeing your comments, I realize that you (and the team) has not yet realized the terrible(!) quality of your documentation.

As I cannot invest time in "Fighting over details", here is the thing:

Either you accept that the documetation needs an overall rework, and that this needs to be outsourced to a person that you trust to produce better quality documentation, or you continue to produce docs by yourself.

As for your comments:

* You don't use CAPITALS to for emphasis.
* The LICENSING text is forced upon the reader in different locations (repetition)
* Stale stuff is usually detected if a newcomer processes the overall documentation (a fulltime task)

So, do you realize that your docs are of terrible quality or not?

If you write even one word of justification now, than it's a "no", and I back off and process your docs only to do my own evaluation of holochain (e.g. not trying to refine the docs)

**Comment by thedavidmeister** [comment]
np, if discussion = fighting then let's close this one out

**Pull request closed without merging**